config-example.yml

```diff
@@ -268,6 +268,10 @@ extensions:
   # thing to have on, but it is possible to turn it off.
   # Default: true
   graphqlScalarExamples: true
+  # It defines from which subQuery depth, the subqueries should be collapsed as a Fragments in preview
+  # instead of showing all properties.
+  # Default: 1
+  collapseSubQueryDepth: 3
```

Comment:
I think this should be named something else like `typeExpansionDepth`.

And I think it should be placed under `introspection` options, and not the `extensions` area.